.cicd/create-docker-from-binary.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+set -euo pipefail
+
+buildkite-agent artifact download '*.deb' --step ':ubuntu: Ubuntu 18.04 - Package Builder' .
+echo ":done: download successful"
+
+SANITIZED_BRANCH=$(echo "$BUILDKITE_BRANCH" | sed 's.^/..' | sed 's/[:/]/_/g')
+SANITIZED_TAG=$(echo "$BUILDKITE_TAG" | sed 's.^/..' | tr '/' '_')
+echo "$SANITIZED_BRANCH"
+echo "$SANITIZED_TAG"
+
+# do docker build
+echo ":docker::build: Building image..."
+DOCKERHUB_REGISTRY="docker.io/eosio/eos"
+
+DOCKER_BUILD_GEN="docker build -t eos_image  -f ./docker/dockerfile ."
```

Comment:
One really quick thing we should take care of now and we'll get this merged today. With our buildfleet concurrency setup, there is a non-zero chance that we could have multiple builders using the same Docker daemon and creating images with this same name at the same time. This could cause build failures randomly.

Let's quickly update this logic to support our CICD and local runs if someone wanted. Update to look like this so that if we are running in Buildkite, we can tag appropriately. Otherwise, it will tag with latest.
```
BUILD_TAG=${BUILDKITE_BUILD_NUMBER:-latest}
DOCKER_BUILD_GEN="docker build -t eos_image:$BUILD_TAG -f ./docker/dockerfile ."
```

Reply:
Done Scott, that is a really important matter, thanks for the information good to know